app/javascript/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import { Message } from './Message';
+import { Link } from 'react-router-dom';
+
+const App = () => {
+  return (
+    <Provider store={store}>
+        <div className="App">
+            <BrowserRouter>
+            <Link to="roots/messages">Go page details</Link>
+              <Routes>
+                <Route path="roots/messages" element={<Message />} />
+              </Routes>
+            </BrowserRouter>
+        </div>
+    </Provider>
+  )
```

Comment:
- [ ] Kindly make sure that your app is running properly. Currently, when I run the app, I get the following error which prevents me from trying your app.
![image](https://user-images.githubusercontent.com/62290545/207935331-b6bdfcf7-e317-475c-9ee7-e92bde37b19d.png)

- [HINT] This error is caused by `cors`, you can read more about it, how it works, and how to fix it from [here](https://www.stackhawk.com/blog/rails-cors-guide/)

Reply:
Well resolved 👍🏽